app/controllers/claims/schools_controller.rb

```diff
@@ -11,9 +11,9 @@ def show
 
   private
 
-  def redirect_to_school_when_belongs_to_one_school
+  def redirect_to_school_claims_when_belongs_to_one_school
     if policy_scope(Claims::School).one?
-      redirect_to claims_school_path(current_user.schools.first)
+      redirect_to claims_school_claims_path(current_user.schools.first)
```

Comment:
The prototype redirects the user to the claims index page, not the school details page